test-e2e/fixtures/zk_scripts/dcos-zk-backup.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
Let's discuss tomorrow converting this to Python. That might be Python which does a bunch of work in `subprocess.run`.